core/includes/install.core.inc

```diff
@@ -194,6 +200,10 @@ function install_state_defaults() {
     // A list of installation tasks which have already been performed during
     // the current page request.
     'tasks_performed' => array(),
+    // The server URL where the interface translation files can be downloaded.
+    // Tokens in the pattern will be replaced by appropriate values for the
+    // required translation file.
+    'server_pattern' => 'https://localize.backdropcms.org/files/l10n_packager/%core/%project/%project-%version.%language.po',
```

Comment:
Maybe the first part of the URL should be a constant too so as to keep it all in one place.

Reply:
I dont quite understand this comment @herbdool

Reply:
@herbdool TBH, I'm in the same boat as @docwilmot here...
Do you suggest to split this URL into chunks? Sounds confusing to me.

Reply:
I'm going to revise my suggestion: move the whole translation file pattern to a constant. It could look like:

```
/**
 * The URI used to fetch the list of available translations from the
 * localization server.
 */
define('INSTALL_LOCALE_FILE_PATTERN', 'https://localize.backdropcms.org/files/l10n_packager/%core/%project/%project-%version.%language.po');
```

Then you can also get rid of the second parameter in `install_check_translation_download` so that it's just `install_check_translation_download($langcode)`. This way we stick to the pattern where these URLs are not buried in the code, which makes it easier to update in the codebase future if we have to. (Also see `update.module` where `UPDATE_DEFAULT_URL` is a constant.) And it recognizes that this is not "variable"; it doesn't need to change during runtime.

Reply:
Done